Added pre-requisite (#3549)
mattermost/docs: source/deployment/auth.rst

Supported smartphones include iOS, Android, Blackberry and Windows Phone devices that are able to install `Google Authenticator <https://support.google.com/accounts/answer/1066447?hl=en>`__. Other than requiring internet access to download and install Google Authenticator, the phone used for Mattermost MFA does not require internet access.

> Pre-requisite

Reviewer: ```suggestion
Prerequisite
```

-------------

> Ensure that your server system time is accurate as the MFA is a time-based system. 

Reviewer 2: ```suggestion
Ensure that your server system time is accurate as the MFA implementation is relying on Time-based One-time passwords (TOTP).
```

> Ensure that your server system time is accurate as the MFA implementation is relying on Time-based One-time passwords (TOTP).

Reviewer 3: ```suggestion
Ensure that your server system time is accurate as the MFA implementation relies on Time-based One-time passwords (TOTP).
```
